ingestion/src/metadata/workflow/base.py

```diff
@@ -114,6 +122,10 @@ def ingestion_pipeline(self) -> Optional[IngestionPipeline]:
 
         return self._ingestion_pipeline
 
+    def with_output_handler(self, output_handler: WorkflowOutputHandler):
+        self._output_handler = output_handler
+        return self
```

Comment:
We could also use a setter method if we are going to keep it as a `property`

Reply:
I think it makes sense. We can change it to a setter without returning self.